plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/EditConnectionWizard.java

```diff
@@ -294,6 +320,11 @@ protected PersistResult persistDataSource() {
         }
     }
 
+    private boolean isDriverLibsVersionsSame(@NotNull DataSourceDescriptor dsChangedSource) {
+        Map<String, String> currentLibs = getLibsIdVersion(dsChangedSource);
+        return Objects.equals(originalDriverLibsIdVersion, currentLibs);
```

Comment:
Instance `equals` can be used instead

```suggestion
        return originalDriverLibsIdVersion.equals(currentLibs);
```